src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -81,13 +82,15 @@
                     "RequireEmptyLineBeforeBlockTagGroup",
                     "SingleLineJavadoc",
                     "SummaryJavadoc",
-                    "WriteTag"
+                    "WriteTag",
+                    "JavadocParamOrder"
```

Comment:
This is not a javadoc check until it extends `AbstractJavadocCheck`.

Reply:
i was just trying to make CI test green but i get your point.

Reply:
i have a question 
JavadocMethod also extends AbstractCheck but it's in here as well( line 73 ) and not just this there are more.

Reply:
Ok, I can't explain it. This item becomes mute when the first item is done. Based on the documentation given for this field, it should be for only pure Javadoc checks. I can only assume normal checks were added to this that should eventually be converted to Javadoc.